src/globalComponents/userAPIIntegration.js

```diff
@@ -1,24 +1,51 @@
 //@flow strict
 
 import Cookies from 'universal-cookie';
+import getLoginToken from '../globalComponents/getLoginToken.js';
 
 /*
 	This function has no input
 	This function returns a json object of the user
 */
-function getUser() : Promise<Response> {
+function getUser(): Promise<Response> {
```

Comment:
nit: we should make this use the same format that everything else does

Reply:
With your getUserAPICall, this is technically no longer necessary. Or can be replaced.